MSAL/src/native_auth/public/state_machine/delegate/SignUpDelegates.swift

```diff
@@ -28,21 +28,22 @@ import Foundation
 public protocol SignUpPasswordStartDelegate {
     /// Notifies the delegate that the operation resulted in an error.
     /// - Parameter error: An error object indicating why the operation failed.
-    @MainActor func onSignUpPasswordError(error: SignUpPasswordStartError)
+    @MainActor func onSignUpPasswordStartError(error: SignUpPasswordStartError)
 
     /// Notifies the delegate that a verification code is required from the user to continue.
+    /// - Note: If a flow requires this optional method and it is not implemented, then ``onSignUpPasswordError(error:)`` will be called.
```

Comment:
This is now incorrect because you renamed the error method! Should say "...onSignUpPasswordStartError(error:)..."